tests/build_tests/super_errors/expected/fun_return_poly1.res.expected

```diff
@@ -19,5 +19,5 @@
   [1;31m4[0m [2m│[0m let err = f(1, [1;31m2[0m)
   5 [2m│[0m 
 
-  The function applied to this argument has type ('a, ~def: int=?) => 'b
-This argument cannot be applied without label
\ No newline at end of file
+  The argument at this position should be labelled.
+  This function has type: ('a, ~def: int=?) => 'b
```

Comment:
We still have different styles here. Other messages have

1. function type
2. error message

This one has

1. error message
2. function type

Reply:
Well, technically I'd say we have this structure:
1. "Header"
2. Function type
3. Error message

For this particular error message I think it makes more sense to have it like this. You're going to see the highlighted position first, then the message that the arg should be labelled, and then the function type. I think that makes sense.

Reply:
Ok, fine with me, too.